tools/cfn2ts/lib/index.ts

```diff
@@ -17,8 +17,6 @@ export default async function(scope: string, outPath: string, force: boolean) {
     const generator = new CodeGenerator(name, spec);
 
     if (!force && await generator.upToDate(outPath)) {
-        // tslint:disable-next-line:no-console
-        console.log('Generated code already up-to-date: %s', colors.green(path.join(outPath, generator.outputFile)));
```

Comment:
This might be a nice hint to stop yourself from going insane if you're updating the `cfn2ts` source and yet no changes are appearing. Can we have it at debug level?

Reply:
I'll revert for now. We don't have support for log levels there at the moment.